models/azure_project.rb

```diff
@@ -587,7 +587,10 @@ def update_bearer_token
   end
 
   def get_prices
-    regions = InstanceLog.where(host: "Azure").select(:region).distinct.pluck(:region).sort
+    regions = InstanceLog.where(host: "Azure").select(:region).distinct.pluck(:region)
+    regions << "uksouth" if !regions.include?("uksouth")
```

Comment:
If you want to be fancy and use some set theory, you could write `regions | ["uksouth"]` here (or even add `| ["uksouth"]` to the above query.